cmd/put.go

```diff
@@ -90,11 +99,16 @@ func putS3Cse(bucket string, key string, kmskeyid string, source string) error {
 	// Create an encryption and decryption client
 	svc := s3crypto.NewEncryptionClient(sess, s3crypto.AESCBCContentCipherBuilder(handler, crosscrypto.NewPKCS7Padder(16)))
 
-	_, err = svc.PutObject(params)
+	result, err := svc.PutObject(params)
+
 	if err != nil {
-		return fmt.Errorf("bad response: %s", err)
+		fmtErr := fmt.Errorf("bad response: %s", err)
+
+		return nil, fmtErr
+	} else {
```

Comment:
Instead of ignoring a possible error with the json.Marshal - what about doing another assignment and if statement?
```

result, err := svc.PutObject(params)

if err != nil {
  fmtErr := fmt.Errorf("bad response: %s", err)
  return nil, fmtErr
}

versionId, err := json.Marshal(result.VersionId)

if err != nil {
  fmtErr := fmt.Errorf("Issue with json.Marshal %s", err)
  return nil, fmtErr
}

return versionId, nil
```